src/Microsoft.AspNet.OData.Shared/HttpMethodHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// A helper class for comparing Http methods.
+    /// </summary>
+    internal class HttpMethodHelper
+    {
+        /// <summary>
+        /// "Get"
+        /// </summary>
+        public const string HttpGet = "GET";
```

Comment:
>public const  [](start = 8, length = 13)

public const are kinda risky, want to use a static readonly instead?

Reply:
This change is a bit harder due to these constants being used in switch/case statement. I push a change for all other comments, still working on this one.